### PR TITLE
Add meta dir for next lesson and edit the readme

### PR DESCRIPTION
**Pull request by SarahAlidoost**

### CODE REVIEW

**Comment by SarahAlidoost** [comment]
@jhidding and @mkuzak please have a look at this PR when you have time and let me know if we can merge it.

File: next-lesson/advertisement.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div style="background: red; color: black;">FIXME</div>
+
+Please delete the contents of this file and fill it with your own
+
+## Expected contents
+
+Two sentences for in advertisement email
```
Comment (mkuzak) [inline next-lesson/advertisement.md:7]:
```suggestion
Two sentences for an advertisement email
```

File: next-lesson/description.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div style="background: red; color: black;">FIXME</div>
+Please delete the contents of this file and fill it with your own
+
+## Expected contents
+
+~150 words for on Eventbrite page
```
Comment (mkuzak) [inline next-lesson/description.md:6]:
```suggestion
~150 words for on Eventbrite registration page
```

File: next-lesson/schedule.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<div style="background: red; color: black;">FIXME</div>
+
+Please delete the contents of this file and fill it with your own
+
+## Expected contents
+
+The times and what you will be teaching at each day in a time-table format.
```
Comment (mkuzak) [inline next-lesson/schedule.md:7]: It would be nice to prefill it with the table with an example 2-day schedule.

File: next-lesson/setup.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+Please delete the contents of this file and fill it with your own
+
+## Expected contents
+
+A list of the requirements.
+
+## Example
+
+- For most users we recommend that you use `conda` to install the requirements for the workshop.
```
Comment (mkuzak) [inline next-lesson/setup.md:11]: Add information that the setup should cover 3 major platforms (WIndows, MacOS and Linux).

File: next-lesson/synopsis.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<div style="background: red; color: black;">FIXME</div>
+
+Please delete the contents of this file and fill it with your own
+
+## Expected contents
+
+140 characters for Eventbrite
```
Comment (mkuzak) [inline next-lesson/synopsis.md:7]:
```suggestion
140 characters for the Eventbrite registration page.
```

**Review by mkuzak — CHANGES_REQUESTED** [review]
Thank you @SarahAlidoost I really like this template. I added a few change suggestions.

File: next-lesson/schedule.md (hunk 3 above)
Comment (SarahAlidoost) [reply next-lesson/schedule.md:7]: good idea, it is added.

File: next-lesson/setup.md (hunk 4 above)
Comment (SarahAlidoost) [reply next-lesson/setup.md:13]: it is added.

**Review by mkuzak — APPROVED** [review]

**Pull request merged**